scripts/globals/crafting.lua

```diff
@@ -297,15 +297,16 @@ function unionRepresentativeTrade(player, npc, trade, csid, guildID)
         if remainingPoints == 0 then
             player:messageText(npc, NO_MORE_GP_ELIGIBLE);
         else
+            local totalItems = 0;
             local totalPoints = 0;
             for i=0,8,1 do
```

Comment:
`for i = 0, 8 do
end`

the trailing 1 is unnecessary. It's the default increment.